packages/frontend/src/components/modal/TxStatusModal.tsx

```diff
@@ -49,16 +53,45 @@ function TxStatusModal(props: Props) {
       />
 
       <Box display="flex" alignItems="center" className={styles.txStatusInfo}>
-        <Typography variant="body1">
-          {(tx && tx.token && timeEstimate) ? (
-            <em>
-              Your transfer will arrive at the destination in <strong>{timeEstimate}</strong>{' '}
-              after your transaction is confirmed.
-            </em>
+        <Box>
+          {(tx && tx.token && medianTimeEstimate) ? (
+            <Box margin="0 auto" maxWidth="32rem" paddingLeft={3} paddingRight={3}>
+              <Typography variant="body2" color="textSecondary">
+                Your {tx.token._symbol ? tx.token._symbol : "transfer"}{' '}
+                will arrive{' '}
+                {
+                  tx.destNetworkName
+                  ? `on ${networkSlugToName(tx.destNetworkName)}` 
+                  : "at the destination"
+                }
+                <strong>{' '}~
+                  {(medianTimeEstimate !== null && medianTimeEstimate > 0) 
+                  ? (medianTimeEstimate + " " + `${pluralize('minute', medianTimeEstimate)}`) 
+                  : (fixedTimeEstimate + " " + pluralize('minute', fixedTimeEstimate))}
```

Comment:
suggestion: move this logic to a util function to make the jsx more readable

```ts
function transferTimeDisplay(estimates) {
 const { mediaTimeEstimate, fixedTimeEstimate } = estimates
 return (
   medianTimeEstimate !== null && medianTimeEstimate > 0) 
                  ? (medianTimeEstimate + " " + `${pluralize('minute', medianTimeEstimate)}`) 
                  : (fixedTimeEstimate + " " + pluralize('minute', fixedTimeEstimate)
_
}
````

```ts
{transferTimeDisplay({ mediaTimeEstimate, fixedTimeEstimate )}
```